haw-pr.jenkinsfile

```diff
@@ -0,0 +1,108 @@
+                                '-Dtycho.localArtifacts=ignore',
+                                // for adjusting the script only, renable later on:
+                                 '-Dmaven.test.failure.ignore',
+                                 "--fail-at-end",
+                                '-DWORKSPACE=' + env.WORKSPACE,
+                                '-DexcludeJRE'                               
+                            ].join(' ')
+                            def profiles = [
+                                'buildProduct',
+                                'execute-plugin-tests',
+                               // 'execute-plugin-ui-tests',
+                                'execute-ecma-tests',
+                                'execute-accesscontrol-tests',
+                                'execute-smoke-tests'
+                             //'execute-hlc-integration-tests'
+                            ].join(',')
+
+                            sh """\
+                                pwd
+                                git log -n 1
+                                npm version
+                            """
+                            sh "mvn ${envvars} clean verify -P${profiles} ${options}"
+                            // works:
+                            // sh "mvn clean verify ${options}"
+                            // for quick testing: 7
+                            // sh "mvn -o clean"
+
+                          // sh "ls -Ral builds/org.eclipse.n4js.product.build/target/repository/"
+                        }
+                    // } // end dir
+                } // end wrap
+            } // end steps
+        } // end stage
+        /*
+        stage('Deploy') {
+            steps {
+                echo 'Deploying....'
+            }
+        }*/
+    }
+
+    post {
+        always {
+         //   archiveArtifacts allowEmptyArchive: true, artifacts: '**/builds/**/target/products/*.zip'
+         //   archiveArtifacts allowEmptyArchive: true, artifacts: '**/tools/**/target/n4jsc.jar'
+         //   archiveArtifacts allowEmptyArchive: true, artifacts: '**/logs/*.log'
+         //   archiveArtifacts allowEmptyArchive: true, artifacts: '**/tests/**/target/**/*-output.txt'
+
+            junit '**/surefire-reports/**/*.xml'
+            //junit '**/failsafe-reports/**/*.xml'
+        }
+        cleanup {
+            // excute here in case archiving fails in 'always'
+            //mail to: 'some.one@some.where.eu',
+            //     subject: "${currentBuild.result}: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]'",
+            //     body:  """\
+            //            ${currentBuild.result}: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]':
+            //            Check console output at '${env.BUILD_URL}'
+            //            """
+
+            // Execute after every other post condition has been evaluated, regardless of status
+            // See https://jenkins.io/doc/book/pipeline/syntax/#post
```

Comment:
Delete? (Also last line below)

Reply:
yes. I will adjust the clean up once the build is working stable.